libs/payments/ui/src/lib/client/components/SubscriptionContent/en.ftl

```diff
@@ -10,8 +10,6 @@
 ## $taxDue (Number) - The tax added on, not included in amount. It will be formatted as currency.
 
 subscription-content-coupon-will-be-applied = { $promotionName } discount will be applied
-subscription-content-heading-cancel-subscription = Cancel Subscription
-subscription-content-no-longer-use-message = You will no longer be able to use { $productName } after { $currentPeriodEnd }, the last day of your billing cycle.
 subscription-content-cancel-access-message = Cancel my access and my saved information within { $productName } on { $currentPeriodEnd }
```

Comment:
This was moved to `libs/payments/ui/src/lib/client/components/CancelSubscription/en.ftl` but not removed from this file.

Reply:
Removed, thank you!